source/documentation/contributing_to_docs/contributing.adoc

```diff
@@ -0,0 +1,167 @@
+
+// * Submit comments at the bottom of each topic (still awaiting implementation)
+// * Email the oVirt documentation team rhev-docs@redhat.com
+* https://github.com/oVirt/ovirt-site/issues/new[Create an issue in GitHub]
+* Submit a PR. You can clone the repository, make your changes, and submit a PR.
+This option is best if you have substantial changes. If you open a PR, be sure
+that all of its contents are related and apply to the same versions.
+
+.What happens when you submit a PR?
+
+The
+// https://github.com/orgs/openshift/teams/team-documentation[documentation team]
+documentation team
+reviews the PR and arranges further review by the development and quality
+assurance teams, as required.
+If the PR requires changes, updates, or corrections, we will let you know
+in the PR. We might request that you make the changes, or let you know that we
+incorporated your content in a different PR. When the PR has been reviewed, all
+updates are complete, and all commits are squashed, we'll merge your PR and
+apply it to the valid versions.
+
+== Repository organization
+The oVirt documentation repository is stored under `/source/documentation`. Each directory corresponds to a guide, except `common`, which includes files that are shared between multiple guides. Each guide's directory includes `index.adoc` and `master.adoc` files. These are the top-level assemblies for that guide, where `index.adoc` is for oVirt and `master.adoc` is for Red Hat Virtualization, a downstream distribution of oVirt. Many guides include additional nested assemblies and a `topics` subdirectory with individual topics.
+
+== Version management
+Most of the content applies to oVirt and Red Hat Virtualization. While a large
+amount of content is reused for both products, some information
+applies to only one or the other. Content inclusion and exclusion is managed
+by using a combination of `include` statements, `ifdef/endif` statements, and attributes
+in individual files.
+// by specifying distributions in the
+//`&#95;topic&#95;map.yml` file or by using `ifdef/endif` statements in individual files.
+
+////
+While it is _possible_
+to accomplish this solely with Git branches to maintain slightly different
+versions of a given topic, doing so would make the task of maintaining internal
+consistency extremely difficult for content contributors.
+
+////
+
+=== Conditional text between products
+oVirt documentation uses AsciiDoc's `ifdef/endif` and `ifeval/endif` macros to conditionalize
+and reuse content, down to the single-line level.
+
+The supported distribution attributes used with the oVirt build mechanism
+are:
+
+* _ovirt-doc_
+* _rhv-doc_
+
+Here is an example of this concept in use:
+
+----
+This line is not conditionalized, and will appear for both oVirt and RHV.
+
+\ifdef::ovirt-doc[]
+This line will only appear for oVirt.
+\endif::[]
+
+ifdef::rhv-doc
+This line will only appear for Red Hat Virtualization.
+\endif::[]
+
```

Comment:
Suggestions remove extra line (line 70). If this is supposed to mimic an onscreen message, add [source,terminal] to line 59 (that is the line above ----)

Reply:
This file is markdown, not AsciiDoc, so that wouldn't work.